### DISCUSSION TITLE
I can translate to spanish...

### DISCUSSION THREAD

**Discussion by gerrom** [albar965/littlenavmap#1264] [General]
**Thank your to contributing to _Little Navmap_.**

**Please take a minute and read the instructions below to help me to avoid unnecessary work and additional communication overhead when reporting a bug or requesting new features.**

All issues for _atools_, _Little Navconnect_, _Little Xpconnect_ are collected in this repository since ownership is often not clear.

Note that almost all _Little Navmap_ dialogs allow to copy formatted text. Simply click and drag the mouse in the dialog to mark the text. Use `Ctrl+C` or context menu to copy to the clipboard. Make screenshots only if relevant for the issue like drawing errors.

Please note that this project is released with a [Contributor Code of Conduct][code-of-conduct]. By participating in this project you agree to abide by its terms.

**Please read before writing an issue**

## If you'd like to report an error:

* Try the steps in the [User Manual - Troubleshoot](https://www.littlenavmap.org/manuals/littlenavmap/release/latest/en/APPENDIX.html#troubleshoot) section of the manual if these solve the problem.
* Check the [list of open issues](https://github.com/albar965/littlenavmap/issues) if you one was already reported. Use the search function.
* Include your OS and version in detail like: `Windows 10 64 bit`, `macOS 10.11` or `Ubuntu Linux 17.10 64 bit`.
* Flight simulator and version like: `Prepar3D v4.1`, `FSX SE`, `X-Plane 11.05` or `X-Plane 11.10.rc2`
* AIRAC cycle and navdata source if applicable: `FSX Stock`, `Navigraph Cycle 1713`, `FSX fsAerodata cycle 1707` or `X-Plane stock data cycle 1611`.
* Attach any related files if copyright and size permits: Flight plan files, FSX/P3D BGL files, or X-Plane apt.dat files. For freeware scenery a link is sufficient.
* Attach the log file and if needed the configuration file. See [User Manual - Files](https://www.littlenavmap.org/manuals/littlenavmap/release/latest/en/FILES.html) for information about file locations.
* Provide all steps to reproduce the error like: Start program with empty database and load flight plan.

## If you'd like propose a new feature:

* Check the list of issues [here](https://github.com/albar965/littlenavmap/issues) and use the search function to check if 
  your proposal is already covered.
* Provide a clear description and attach screenshots, file examples or documentation links that help me to understand and implement the feature.

**Delete this text _after reading_ before posting the issue.**

[code-of-conduct]: https://github.com/albar965/littlenavmap/blob/master/.github/CODE_OF_CONDUCT.md

**Comment by gerrom** [comment]
I can translate to spanish

**Comment by albar965** [comment]
> I can translate to spanish

Thank you for the offer. :+1:  There is already a Spanish translation but I disabled it since it way behind and has other issues.
You can find the file here: https://github.com/albar965/littlenavmap/blob/master/littlenavmap_es.ts

Also see here in the wiki: https://github.com/albar965/littlenavmap/wiki/Translating-the-Program

Do not hesitate to ask since some phrases are not clear.
Alex
